test/e2e/install.spec.js

```diff
@@ -106,11 +106,8 @@ describe('install', () => {
       'one-two.yml': await utils.setVersion('one-two.yml', '13.0.0', false),
     })).to.be.rejected;
     expect(response.error).to.equal(true);
-    expect(response.reason).to.include(
```

Comment:
Newer versions of `docker compose` (and `docker-compose` for that matter...) seem to now only report one of the images that fails to pull (verified this behavior locally).  My guess is that there is some indeterminate race-conditions as to which will missing image will be reported....